test/ansible-operator/cmd/ansible-operator/main.go

```diff
@@ -37,9 +37,10 @@ func printVersion() {
 }
 
 func main() {
-	flag.Parse()
+	aflags := aoflags.AddTo(pflag.CommandLine)
+	pflag.Parse()
 
-	logf.SetLogger(logf.ZapLogger(true))
+	logf.SetLogger(logf.ZapLogger(false))
```

Comment:
The logger should be in development mode so we see all `Debug` level logs in test output.